src/main/java/me/partlysanestudios/partlysaneskies/OneConfigScreen.java

```diff
@@ -202,13 +202,19 @@ public void resetBrokenStrings() {
     )
     public boolean colorCoopChat = false;
 
+    @VigilanceName(
```

Comment:
We'd have to use another annotation.

Essentially, remove the @VigilanceName part, and it's up to you if you want to reset the config for everyone